.vscode/launch.json

```diff
@@ -48,6 +48,15 @@
 			],
 			"internalConsoleOptions": "openOnSessionStart",
 			"preLaunchTask": "npm: compile"
+		},
+		{
+			"type": "scala",
+			"request": "launch",
+			"name": "DAPodil",
```

Comment:
What does this actually launch? What is DAPodil?

Reply:
This is the launch config for the scala backend. I was using this during debugging. I can remove this if we don't want it in the repo, but I think we should at least have it documented somewhere because it is needed to debug the scala backend.

Reply:
What about "Daffodil Debug Server"? I'm not sure if most people will know what a "DAP" server is. Also, if it's only for extension developers, maybe it should be hidden behind an "advanced settings" option or something?

Reply:
You're right it's only for extension developers, and one would only see this if they cloned the repo and were using VS Code, so I think it's ok to be specific about its DAP purpose.

Reply:
> would only see this if they cloned the repo and were using VS Code

How does that work? I don't see anything that only shows this menu for extension developers?

Reply:
Ah, maybe I just had to look up, is this part of the "Extension" configuration? Is that only enabled by developers somehow?

Reply:
The .vscode/launch.json file is used to define runnable programs within the current VS Code project. It doesn't necessarily have anything to do with extension development or not.

Reply:
```suggestion
			"name": "Daffodil Debug Server",
```